### PR TITLE
Add: Catch errors when loading a feed

### LINKED ISSUE #7: Application bricks when channelID is invalid
If there's an import problem (e.g. because you confused channel:name and channelID), the application won't come out of the loading stage.

To do:
* Unblock the UI if there was an error loading the feed.
* Display an helpful error.

### PR DESCRIPTION
**Pull request by jeyemwey**
Closes #7.

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -158,10 +160,16 @@ export default {
...
-      this.loading = false;
-      this.addChannel(feed);
-      this.addVideos(feed);
+      try {
+        const feed = await loadFeed(channelID);
+        this.addChannel(feed);
+        this.addVideos(feed);
+      } catch (e) {
+        this.errorMessage = e.message;
+        setInterval(() => { this.errorMessage = ''; }, 10 * 1000); // Automatic dismiss after 10 seconds
```
Comment (pixeldesu) [inline src/App.vue]: shouldn't this be `setTimeout`? `setInterval` resets it **every 10 seconds**.

Easily replicatable, cause an error, wait for the alert to be dismissed and then wait a few seconds before causing another feed pulling error. The second alert will only stay for a handful of seconds, even less maybe.

Comment (jeyemwey) [reply src/App.vue]: Jup, should be `setTimeout`, my bad. If we're using snackbars, we should also find a way to show several errors at once and let them timeout by themselves.

**Comment by jeyemwey** [comment]
Snackbars look good!

File: src/App.vue (hunk 1 above)
Comment (pixeldesu) [reply src/App.vue]: maybe doing `errorMessages` as an Array, pushing into it to add new errors? If the old ones already were dismissed by `timeout` only new ones should be created anyway.

This way, at some point later in time, we could also show an error log in the frontend, of sorts.

**Comment by pixeldesu** [comment]
I did a major refactoring on the code in the `develop` branch today and moved to a completely stateful model using Vuex.

You need to update your PR and then apply your changes there!

**Comment by pixeldesu** [comment]
Added in 280636b

**Pull request closed without merging**